packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -368,7 +368,7 @@ export interface IContextualMenuSection extends React.Props<ContextualMenu> {
   /**
    * The optional section title.
    */
-  title?: IContextualMenuItem;
+  title?: string;
```

Comment:
This is very much a breaking change, however this API was just introduced last week and probably has no consumers yet.